frontend/lib/store_widgets/detail/contact_info_row.dart

```diff
@@ -44,6 +45,7 @@ class ContactInfoRow extends StatelessWidget {
         Expanded(
           child: Text(
             _description,
+            style: theme.textTheme.bodyMedium,
```

Comment:
not sure if that's important, but it was grey before, while now it's black

Reply:
hm i think thats okay

Reply:
I personally think it looks more appealing when colors are different, but maybe it's too subjective